src/components/Dashboard/TodoList.vue

```diff
@@ -79,7 +79,23 @@
                   <span>{{ task.subtitle }}</span>
                 </div>
 
-                <div v-else-if="isStatusDraft(task) && (isTypeCorrection(task) || isTypeAlteration(task))"
+                <div v-else-if="isStatusDraft(task) && isTypeAlteration(task) && !task.goodStanding"
+                  class="todo-list-detail body-2"
+                >
+                  <p class="red--text">
+                    <v-icon color="red" left>mdi-information-outline</v-icon>
+                    This business is not in good standing.
+                  </p>
+
+                  Before you can alter your business it must be in good standing with the Business Registry.
+                  There may be several reasons a business is not be in good standing, but the most
+                  common reason is an overdue annual report.<br>
+                  To resolve this issue, you MUST contact registries staff:
```

Comment:
Check capitalization of "registries staff" elsewhere in this app. I think it needs to be title cased.

Reply:
changed to standard in app